app/exceptions/atmosphere/unsupported_exception.rb

```diff
@@ -0,0 +1,3 @@
+module Atmosphere
+  class UnsupportedException < Exception; end
+end
```

Comment:
this is correction of very common bug in code I commit - misspell. Instead of `UnsupportedException` class was named `UnsuportedException` (one `p` missing).